lib/extensions/ar_adapter/ar_pglogical.rb

```diff
@@ -0,0 +1,228 @@
+module PgLogical
+  def pglogical_installed?
+    exec_query("SELECT EXISTS(SELECT * FROM pg_available_extensions WHERE name = 'pglogical')").cast_values.first
+  end
+
+  # Enables pglogical postgres extensions
+  def pglogical_enable
+    enable_extension("pglogical")
+    enable_extension("pglogical_origin")
+  end
+
+  # Node Management
+  #
+
+  # Creates a node
+  #
+  # @param name [String]
+  # @param dsn [String] external connection string to the node
+  def pglogical_node_create(name, dsn)
+    raw_connection.exec_params("SELECT pglogical.create_node($1::VARCHAR, $2::VARCHAR)", [name, dsn])
```

Comment:
Do we actually need these casts? And do we need to use raw_connection, or can we use a higher level method?

If we were aiming to consciously design this to only interact with the raw connection, I could understand choosing to put it at that layer... but as we already have some calls that use the AR adapter, it seems logical that we'd use it for everything.